manifest.yml

```diff
@@ -3,6 +3,8 @@
 memory: 1G
 instances: 1
 applications:
-- name: foia
+- name: krang
   path: .
   timeout: 180
+  services:
+  - es
```

Comment:
On the app side, yes; however, there are a couple of steps that devops takes care of before [listed here](https://docs.cf.18f.us/ops/accessing-v1-managed-services/)